src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/commands.py

```diff
@@ -71,6 +71,8 @@ def load_command_table(self, _):
     with self.command_group('webapp', webapp_sdk) as g:
         g.custom_command('create', 'create_webapp', exception_handler=ex_handler_factory())
         g.custom_command('list', 'list_webapp', table_transformer=transform_web_list_output)
+        g.custom_command('list-deleted', 'list_deleted_webapp')
```

Comment:
For "az webapp deleted show" it will be possible for multiple deleted apps to be returned, because a webapp with the same name could be deleted and re-created multiple times. Is it ok for a "show" command to return more than 1 resource?

Reply:
then just do `az webapp deleted list/restore`.